onnxruntime/core/providers/ngraph/ngraph_custom_op.cc

```diff
@@ -84,9 +84,19 @@ void NGRAPHCustomOp::Initialize(const OrtCustomOpApi* api, OrtKernelContext* con
 
   // Get cache size from environment
   std::string tempSize;
+  #ifdef _WIN32
+  char *buf{nullptr};
+  size_t bufSize;
+  _dupenv_s(&buf, &bufSize, "ONNXRUNTIME_NGRAPH_LRU_CACHE_SIZE");
+  if(buf) {
+    tempSize = buf;
+  }
+  free(buf);
```

Comment:
should only free if _dupenv_s() returns 0 && buf != nullptr

Reply:
Fixed.